DFC.ServiceTaxonomy.GraphSync/GraphSyncers/Helpers/GraphSyncHelper.cs

```diff
@@ -163,7 +163,7 @@ public async Task<string> GenerateIdPropertyValue()
                 $"http://data.europa.eu/esco/{_contentType!.ToLowerInvariant()}/{newGuid}",
 
                 "$\"<<ContentApiPrefix>>/{ContentType.ToLowerInvariant()}/{Value}\"" =>
-                $"{_contentApiBaseUrl}/{_contentType!.ToLowerInvariant()}/{newGuid}",
+                $"{_contentApiBaseUrl}/{_contentType!.ToLowerInvariant()}/{newGuid}".ToLowerInvariant(),
```

Comment:
The config value and what the optimization creates should match, so we need to change the recipe.
(So any ToLower needs to live within the config value).

Reply:
This is only called when you create a new Content Item from inside the editor. ToLowerInvariant() is called to make sure the config value is lowered